### ISSUE TITLE
Issue with OutputOnlyGenotypedAnimals option

### ISSUE THREAD

**Issue by Agustinopulus** [AlphaGenes/AlphaImpute2#6]
Dear AlphaGenes team
   Im using Alphaimpute2, to impute SNP array data ( ~ 60K SNPs) from offspring  to WGS (~ 7M SNPs) to the parents . Also I have a pedigree file from 15 generations back (~ 85 K animals).
My issue is that the generated output has imputed data from all the fish, included those on the pedigree. However I only require the imputation from the genotyped animals.
On the Alphaimpute v.1.9 manual (I cant find the manual for the v2.0) there is an option called Outputonlygenotypedanimals. I’ve tried this option but it looks like it does not work for the last version of alphaimpute (at least for me). I’ve tried not using the pedigree for the imputation, and it works in terms that I only have the output for the offspring. However, my idea is to use the pedigree information as well, to improve the accuracy of the analysis. 
It is possible to use the "Outputonlygenotypedanimals" on the v2? I wrote this option on the last box (BOX 8; Output), as shown below:
= BOX 8: Output =============================================================
WellPhasedThreshold         ,99.0
ResultFolderPath            ,Results
OutputOnlyGenotypedAnimals  ,Yes

I appreciate your help

Cheers

**Comment by gregorgorjanc** [comment]
Hi @Agustinopulus would you care to share what solved the issue? Thanks!

**Comment by Agustinopulus** [comment]
Hi @gregorgorjanc 
   I figured it out that Alphaimute 2.0 doesn't recognize the parameter file necessary for the v 1.9. ( I realized this after trying several changes on the parameter file, e.g. imputation and phasing particles). Therefore, is not yet possible to output only the genotyped animals on Alphaimpute 2.0. Rather, the output shows all the animals, even those included in the pedigree that are not genotyped. 
In my case removed the not-genotyped animals through the UNIX command line.
